src/backend/db/geese-features.js

```diff
@@ -0,0 +1,74 @@
+import * as R from 'ramda';
+
+const getFeatures = (db) => () =>
+  db('geese-features')
+    .then((features) => {
+      return { success: 'Y', features };
+    })
+    .catch((error) => {
+      return { success: 'N', features: [] };
```

Comment:
You can simply return `features` for the .then case and throw an error in the .catch case when getting all features. An empty array is still a valid response from knex.js in this case, but an error should be made explicit to the backend so it can be properly handled.